app/vep/utils/web_metadata.py

```diff
@@ -0,0 +1,26 @@
+from loguru import logger
+import requests
+from typing import List
+from core.config import ENSEMBL_WEB_METADATA_API
+from vep.models.client_model import GenomeAnnotationProvider
+
+async def get_genome_metadata(genome_id: str) -> GenomeAnnotationProvider:
+
+    try:
+      session = requests.Session()
+      with session.get(
+          url=f"https://staging-2020.ensembl.org/api/metadata/genome/{genome_id}/details"
```

Comment:
Part of this url string should come from an environment variable

Reply:
Yes it was! but changed it for testing and forgot to undo. 👍